style.css

```diff
@@ -1,26 +1,45 @@
-.follow-nature li {
-  position: relative;
+.follow-nature ul {
+    overflow: visible;

```

Comment:
the stylesheet that came with the test had set this to `overflow: hidden` which was hiding the portion of the black tooltip that was "overflowing" the `<ul>` container. therefore under my approach i wanted to set this to `overflow: visible` so we can see the entire tooltip. however this meant that the age old problem of a container not surrounding floated elements - which needed a clearfix solution as you'll see in the other comment.